src/AddTodoForm.js

```diff
@@ -1,10 +1,21 @@
 import React from "react";
 
-const AddTodoForm = () => {
+
+const AddTodoForm = (props) => {
+
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        let todoTitle = event.target.title.value;
```

Comment:
nit: you can use `const todoTitle = ...` here instead since you won't be changing the value of `todoTitle`